### ISSUE TITLE
aux path appears reserved

### ISSUE THREAD

**Issue by rsegnini** [razvanmarinescu/EB1A#9]
The clone script failed when copying the files in the aux directory.
I tried creating the directory manually, and got this message:

mv : Cannot process path '.\EB1A\aux' because the target represents a
reserved device name.
At line:1 char:1
+ mv .\aux1\ aux
+ ~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (.\aux:String) [Move-Item], IOException
    + FullyQualifiedErrorId : MoveError,Microsoft.PowerShell.Commands.MoveItemCommand

BTW, thank you for sharing your experience

**Comment by razvanmarinescu** [comment]
you mean when you clone with git clone? as in:
```
git clone git@github.com:razvanmarinescu/EB1A.git
```

**Comment by rsegnini** [comment]
yes. The first time with that exact command line it appeared to start working, then made another error with a similar reason than I already posted.
Yet still nothing was cloned in my local directory, until I wrote:

git restore --source=HEAD :/ 

and that created all the subdirectories, except the one called aux, and provided again the same error message.
By troubleshooting I determined that somehoe aux is a reserved device name.

BTW, I am running Powershell in Windows 11

**Comment by razvanmarinescu** [comment]
Just tested on MacOS and on Linux and it works, it might be something Windows specific. Try copy-pasting the latex code in your latex file. 

<img width="746" alt="Screen Shot 2024-02-28 at 9 58 44 AM" src="https://github.com/razvanmarinescu/EB1A/assets/2820475/10445ab9-7b89-4da8-909f-45d61e96c553">

**Comment by rsegnini** [comment]
Thank you for taking the time. Should have given you all the screens.
Ultimately will do as you say, as I got the latex file. 
I just didn't know how critical for the compile where the files in the aux subdir.

For completeness, let me redo everything.
First, I renamed the directory that I was able to download last time (everything but the aux subdir)
Now I run the git command:

> git clone git@github.com:razvanmarinescu/EB1A.git
Cloning into 'EB1A'...
remote: Enumerating objects: 80, done.
remote: Counting objects: 100% (80/80), done.
remote: Compressing objects: 100% (61/61), done.
remote: Total 80 (delta 30), reused 68 (delta 18), pack-reused 0Receiving objects:  73% (59/80), 14.21 MiB | 1024.00 KiB/s
Receiving objects: 100% (80/80), 14.45 MiB | 1.07 MiB/s, done.
Resolving deltas: 100% (30/30), done.
error: invalid path 'aux/RazvanMarinescu_CV.pdf'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

after doing this, the directory is created empty.

> cd EB1A
EB1A> ls
(nothing shows, prompt comes back)

So I get into it and run the retry command, as recommended:

EB1A> git restore --source=HEAD :/
error: invalid path 'aux/RazvanMarinescu_CV.pdf'
error: invalid path 'aux/adevarul_translation.pdf'
error: invalid path 'aux/adevarul_wiki.pdf'
error: invalid path 'aux/ai_impact1.pdf'
error: invalid path 'aux/ai_impact2.pdf'
error: invalid path 'aux/alzforum_wiki.pdf'
error: invalid path 'aux/check.jpeg'
error: invalid path 'aux/esi.pdf'
error: invalid path 'aux/esi.png'
error: invalid path 'aux/google_scholar.pdf'
error: invalid path 'aux/gscholar_discipline.pdf'
error: invalid path 'aux/tadpole_press.pdf'
EB1A> ls

    Directory: EB1A

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         2/28/2024   1:19 PM                articles
d-----         2/28/2024   1:19 PM                letters
d-----         2/28/2024   1:19 PM                reviews
-a----         2/28/2024   1:19 PM         619984 ETA9089.pdf
-a----         2/28/2024   1:19 PM         744725 i-140_final.pdf
-a----         2/28/2024   1:19 PM        1341854 i-485.pdf
-a----         2/28/2024   1:19 PM         428337 i-907.pdf
-a----         2/28/2024   1:19 PM        8625139 main.pdf
-a----         2/28/2024   1:19 PM         101718 main.tex
-a----         2/28/2024   1:19 PM            990 README.md

Thank you!

**Comment by razvanmarinescu** [comment]
Gotcha. I think it's something with the pdfs. If you manage to fix it, send a pull request and I'll merge it. Meanwhile, just git clone it on a Linux/MacOS system.

**Comment by rsegnini** [comment]
Hi, I don't have those systems. For others, this [link ](https://stackoverflow.com/questions/63727594/github-git-checkout-returns-error-invalid-path-on-windows)has references that can help.
For the future, there are issues with reserved names and characters in Windows.
Possible solution: rename that subdir, or check the link for options.
